timestamp-impl/src/test/java/com/palantir/timestamp/PersistentTimestampServiceTest.java

```diff
@@ -90,7 +91,7 @@ public void incrementUpperLimitIfOneMinuteElapsedSinceLastUpdate() throws Interr
         Thread.sleep(10);
         persistentTimestampService.getFreshTimestamp();
         Thread.sleep(10);
-        verify(timestampBoundStore, times(2)).storeUpperLimit(anyLong());
+        verify(timestampBoundStore, atLeast(2)).storeUpperLimit(anyLong());
```

Comment:
Is there a ticket open to reduce this oddity of it setting the upper limit multiple times?